### ISSUE TITLE
Check if any flavors are missing

### ISSUE THREAD

**Issue by ronaldtse** [relaton/relaton.github.io#46]
I know for example IHO is already missing:
<img width="723" alt="Screenshot 2023-08-18 at 12 35 52 PM" src="https://github.com/relaton/relaton.github.io/assets/11865/3723541e-a9db-4d07-802a-c629af8232ea">

And we need to add them to the Software section:
<img width="816" alt="Screenshot 2023-08-18 at 12 36 22 PM" src="https://github.com/relaton/relaton.github.io/assets/11865/6e8c2891-af91-43ba-a82c-e52af63611f4">

**Comment by ronaldtse** [comment]
And soon CCSDS.

**Comment by andrew2net** [comment]
@ronaldtse I can't understand how to correctly add new flavors to the site. For example, I've added `relaton-callconect` to the `_config.yml`:
```yaml
    ...
    - id: relaton-calconnect
      label: CalConnect
      gem: relaton-calconnect
      sources:
        - { url: "https://www.calconnect.org", label: "CalConnect" }
    ...
```
Then I created `_software/relaton-calconnect.adoc` and copied `_software/relaton-3gpp/assets/symbol.svg` to `_software/relaton-calconnect/assets/symbol.svg`. After generating the site Jekyll creates '_software/relaton-calconnect/.git` dir. When I tried to stage changes I got:
```sh
$ git add .
warning: adding embedded git repository: _software/relaton-calconnect
hint: You've added another git repository inside your current repository.
hint: Clones of the outer repository will not contain the contents of
hint: the embedded repository and will not know how to obtain it.
hint: If you meant to add a submodule, use:
hint: 
hint:   git submodule add <url> _software/relaton-calconnect
hint: 
hint: If you added this path by mistake, you can remove it from the
hint: index with:
hint: 
hint:   git rm --cached _software/relaton-calconnect
hint: 
hint: See "git help submodule" for more information.
warning: adding embedded git repository: _software/relaton-ccsds
```
I didn't find any submodules for other flavors and decided not to add it for `relaton-calconnect`. I removed the `_software/relaton-calconnect/.git` dir and pushed the changes. But now the `_software/relaton-calconnect/assets/symbol.svg` isn't shown on the flavor page:

<img width="464" alt="image" src="https://github.com/relaton/relaton.github.io/assets/5623890/a59459cd-ac11-4696-bd25-f8d67473f782">

Can you give me a direction to address this issue?

**Comment by strogonoff** [comment]
@andrew2net perhaps you have committed some files that were generated during build. I recommend running `make clean` before committing changes to the source. This is due to some quirks of how Jekyll Open Project theme works.

**Comment by strogonoff** [comment]
Broken submodules should definitely  be removed, I see a few that return HTTP 404…

**Comment by ronaldtse** [comment]
I've made the fix for adding flavor icons in #52.

**Comment by ronaldtse** [comment]
I confirm all the new flavors have been added.
